Mathlib/Condensed/Discrete/LocallyConstant.lean

```diff
@@ -186,8 +186,7 @@ def componentHom (a : Fiber (f.comap g)) :
     simp only [Fiber.mk, Set.mem_preimage, Set.mem_singleton_iff]
     convert map_eq_image _ _ x
     exact map_preimage_eq_image_map _ _ a⟩
-  continuous_toFun := by
-    exact Continuous.subtype_mk (Continuous.comp g.continuous continuous_subtype_val) _
+  continuous_toFun := by exact Continuous.subtype_mk (g.continuous.comp continuous_subtype_val) _
```

Comment:
Unrelated to the PR but, can the `by exact` be omitted here?

Reply:
As explained in a comment below, it gives a "free variable error"